modules/mini/src/main/java/org/apache/fluo/mini/MiniFluoImpl.java

```diff
@@ -214,7 +214,7 @@ public static void main(String[] args) {
         System.exit(-1);
       }
       try (MiniFluo mini = new MiniFluoImpl(config)) {
-        log.info("MiniFluo is running");
+        log.info("MiniFluo is running " + mini);
```

Comment:
I looked for Java code and scripts that migh call the main method and did not find any.